packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -37,7 +37,7 @@ enables organizations to create and manage catalogs of products for their end us
 - [Constraints](#constraints)
   - [Tag update constraint](#tag-update-constraint)
   - [Notify on stack events](#notify-on-stack-events)
-  - [CloudFormation parameters constraint](#cloudformation-parameters-constraint)
+  - [CloudFormation template parameters constraint](#cloudformation-template-parameters-constraint)
```

Comment:
Thoughts on `CloudFormation template constraint` instead?

Underlying SC terminology is that it is a Template Constraint and that Parameters are one of the examples a rule can apply to.

https://docs.aws.amazon.com/servicecatalog/latest/adminguide/reference-template_constraint_rules.html

Reply:
I think we've been moving towards more explicit naming.  I'm not actually sure if assertions can apply to anything but the parameters in a meaningful way?